### PR TITLE
ContentHeader and FormWithStepper css update 

### PR DESCRIPTION
**Pull request by Shubhamkumar2023**
- Few Updates done for the ContentHeader and FormwithStepper component.

### CODE REVIEW

**Comment by Shubhamkumar2023** [comment]
FormWithStepper should be visible like as the following - 
image

But it was - 
image

So done the CSS changes to alignment issues and others

File: src/components/Layout/FormPage/FormWithStepper/FormWithStepper.module.scss [hunk 1]
```diff
@@ -19,6 +19,7 @@ $header-height: 13.25rem;
 .wrapper {
   position: relative;
   margin: 0 auto;
+  padding: 2rem 0 0 2rem;
```
Comment (MichalGorskiOnegini) [inline src/components/Layout/FormPage/FormWithStepper/FormWithStepper.module.scss:22]: We don't want that as we prefer to control the spacing in the MicrofrontendContainer component. You can use it in your implementation, if not, you can always add the margins yourself outside of the component.

File: src/components/Layout/ContentHeader/ContentHeader.module.scss [hunk 2]
```diff
@@ -26,9 +26,9 @@
 
 .header-text {
   margin: 0;
-  font-weight: 500;
-  color: var(--color-blue-grey900);
-  line-height: 3.125rem;
+  font-weight: 700;
```
Comment (MichalGorskiOnegini) [inline src/components/Layout/ContentHeader/ContentHeader.module.scss:29]: To be decided by the UX team which of the headers is the latest one :D

**Review by MichalGorskiOnegini — CHANGES_REQUESTED** [review]

File: src/components/Layout/ContentHeader/ContentHeader.module.scss (hunk 2 above)
Comment (cjourdai) [reply src/components/Layout/ContentHeader/ContentHeader.module.scss:29]: From what I see in Figma, it should be:
- font-weight: 500;
- color: var(--color-blue-grey750);
- line-height: 3.125rem;

**Comment by MichalGorskiOnegini** [comment]
Agreed the previous solution was correct.

**Pull request closed without merging**